public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -38,6 +38,18 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 	</div>
 </div>
 
+<div class="gf-form-group">
```

Comment:
This is not used right now so think we can skip this for now. The $__interval should work anyway. Think it's for calculating intervals on backend.